modular_zubbers/code/game/machinery/computer/arcade.dm

```diff
@@ -0,0 +1,157 @@
+#define MINESWEEPER_BEGINNER 1
```

Comment:
Move these defines into the file that uses them, and add #undef at the end of the file that uses them. minesweeper.dm seems to be the only the uses them.